scripts/requirements.in

```diff
@@ -1,2 +1,2 @@
 stac-api-validator
-stac-geoparquet
+stac-geoparquet<0.5  # v0.5 uses geoparquet 1.1, but geoarrow-rs is geoparquet 1.0
```

Comment:
We should probably have an option in stac-geoparquet for whether to write 1.0 or 1.1 compliant metadata.

🥲 I have some support in geoarrow-rs for geoparquet 1.1 but not full support yet.